packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -83,7 +83,7 @@ while getopts "r:c:e:k:d:" option; do
     esac
 done
 
-if [ -z $resourceGroupName ] || [ -z $clientId ] || [ -z $environment ] || [ -z $keyVault ]; then
```

Comment:
How we are planning to get clientId?

Reply:
As discussed offline, either passed or created if it was empty